Helper/OmiseHelper.php

```diff
@@ -111,4 +111,12 @@ public function convertTescoSVGCodeToHTML($svg)
         }
         return $xhtml->saveXML(null, LIBXML_NOEMPTYTAG);
     }
+    /**
+     * Checks if $paymentType is QR payment. This method returns true for QR payment method or else false.
+     * @param string $paymentType
+     * @return boolean
+     */
+    public function isQRCodePayment($paymentType) {
```

Comment:
The function `isQRCodePayment` is confusingly named because `bill_payment_tesco_lotus` is not a QR code payment method.

Reply:
Done. Updated as `isOfflinePayment`.